packages/frontend/src/components/wallet/Wallet.js

```diff
@@ -494,6 +502,7 @@ const FungibleTokens = ({
                             />
                         </>
                     )}
+                    <div className="coingecko"><Translate id="poweredByCoinGecko"/></div>
```

Comment:
nit: let's try to stick with single quotes when possible just for consistency sake